kahuna/public/stylesheets/main.css

```diff
@@ -917,7 +917,7 @@ input.search-query__input {
 
 .results-toolbar-item__metadata-panel-controls {
     line-height: 35px;
-    width: 130px;
+    width: 129.5px;
```

Comment:
Are sub-pixel dimensions a thing? Is that only on retina?

Seems a little odd to me too, but maybe easiest is to show us!